lib/nuget/argu/3.0.0-beta02/argu.nuspec

```diff
@@ -0,0 +1,30 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+  <metadata>
+    <id>Argu</id>
+    <version>3.0.0-beta02</version>
+    <authors>Argu</authors>
+    <owners>Argu</owners>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>Argu</description>
+    <tags></tags>
+    <dependencies>
+      <group targetFramework=".NETFramework4.6">
+        <dependency id="FSharp.Core" version="[4.0.0.1, )" />
+      </group>
+      <group targetFramework=".NETStandard1.6">
+        <dependency id="System.Xml.XDocument" version="[4.0.11, )" />
+        <dependency id="Microsoft.FSharp.Core.netcore" version="[1.0.0-alpha-160629, )" />
+        <dependency id="NETStandard.Library" version="[1.6.0, )" />
```

Comment:
Well usually I'm against squashing as well :). I could very well rework the history in another way to get rid of the binaries, but it's *way* more work ;) . I would say this is an extreme case as it hurts clone speed for no good reason...